internal/k8s/ako_init.go

```diff
@@ -863,64 +863,7 @@ func (c *AviController) FullSyncK8s(sync bool) error {
 		}
 	}
 
-	for namespace := range acceptedNamespaces {
-		svcObjs, err := utils.GetInformers().ServiceInformer.Lister().Services(namespace).List(labels.Set(nil).AsSelector())
-		if err != nil {
-			utils.AviLog.Errorf("Unable to retrieve the services during full sync: %s", err)
-			return err
-		}
-
-		for _, svcObj := range svcObjs {
-			isSvcLb := isServiceLBType(svcObj)
-			var key string
-			if isSvcLb && !lib.GetLayer7Only() {
-				key = utils.L4LBService + "/" + utils.ObjKey(svcObj)
-				if svcObj.Annotations[lib.SharedVipSvcLBAnnotation] != "" {
-					// mark the object type as ShareVipSvc
-					// to separate these out from regulare clusterip, svclb services
-					key = lib.SharedVipServiceKey + "/" + utils.ObjKey(svcObj)
-				}
-			} else {
-				if lib.IsWCP() {
-					continue
-				}
-				key = utils.Service + "/" + utils.ObjKey(svcObj)
-			}
-			meta, err := meta.Accessor(svcObj)
-			if err == nil {
-				resVer := meta.GetResourceVersion()
-				objects.SharedResourceVerInstanceLister().Save(key, resVer)
-			}
-			nodes.DequeueIngestion(key, true)
-		}
-	}
-
-	if lib.GetServiceType() == lib.NodePortLocal {
-		podObjs, err := utils.GetInformers().PodInformer.Lister().Pods(metav1.NamespaceAll).List(labels.Everything())
-		if err != nil {
-			utils.AviLog.Errorf("Unable to retrieve the Pods during full sync: %s", err)
-			return err
-		}
-		for _, podObj := range podObjs {
-			podLabel := utils.ObjKey(podObj)
-			ns := strings.Split(podLabel, "/")
-			if lib.IsNamespaceBlocked(ns[0]) {
-				continue
-			}
-			key := utils.Pod + "/" + podLabel
-			if _, ok := podObj.GetAnnotations()[lib.NPLPodAnnotation]; !ok {
-				utils.AviLog.Warnf("key : %s, msg: 'nodeportlocal.antrea.io' annotation not found, ignoring the pod", key)
-				continue
-			}
-			meta, err := meta.Accessor(podObj)
-			if err == nil {
-				resVer := meta.GetResourceVersion()
-				objects.SharedResourceVerInstanceLister().Save(key, resVer)
-			}
-			nodes.DequeueIngestion(key, true)
-		}
-	}
-
+	// Re-order informer loading. all crds- then objects depends upon it.
```

Comment:
should we take care of this in the 1.11.1 release and not in a patch?

Reply:
Re-ordering is require as Aviinfrasetting can be associated with L4. Vip and node network needs to be populated before L4 processing else those fields will be empty and will result in failure.

Reply:
sure, got it.